specviz/plugins/statistics/statistics_widget.py

```diff
@@ -210,6 +259,38 @@ def clear_statistics(self):
         self._clear_stat_widgets()
         self.stats = None
 
+    def _reconnect_item_signals(self):
+        if self.hub.plot_item is self._current_plot_item:
+            return
+
+        if isinstance(self._current_plot_item, PlotDataItem):
```

Comment:
Just to clarify: is that a particular reason this was necessary? Are you expecting the plot item to be anything else?

Reply:
Nope, this is just a blanket check for `None` and anything that is not a `PlotDataItem` introduced in the future.